test/models/survey_test.exs

```diff
@@ -195,6 +195,7 @@ defmodule Ask.SurveyTest do
       created_survey = Repo.get!(Survey, survey.id) |> Repo.preload([:project, panel_survey: :project])
 
       assert created_survey == survey
+      assert survey.panel_survey_id
```

Comment:
shouldn't we assert is the same as `panel_survey.id` ?

Reply:
Yes, it makes a lot of sense! 🙃